src/cpu/aarch64/jit_primitive_conf.hpp

```diff
@@ -0,0 +1,154 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
```

Comment:
nit: [here and elsewhere] year change in copyright 2020 ---> 2021

Reply:
Thank you. We changed it.